lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -787,6 +799,21 @@ def test_bypass_course_info(self):
         response = views.course_info(request, course_id)
         self.assertEqual(response.status_code, 200)
 
+    def test_accordion(self):
+        table_of_contents = toc_for_course(
+            self.request.user,
+            self.request,
+            self.course,
+            unicode(self.course.get_children()[0].scope_ids.usage_id),
+            None,
+            None
+        )
+
+        view = ''.join(render_accordion(self.request, self.course, table_of_contents['chapters']).split())
+
+        self.assertIn('href="/courses/org.0/course_0/te%EA%9C%B1%E1%B4%9B_course/courseware/Chapter_1/Sequential_1/"><p>Sequential1</p>', view)
```

Comment:
Final comment:
* Can the `course_id` be formatted into the string from `self.course.id` instead of hard-coding it here? 
* By the way, this line is also causing a quality failure.